adafruit_vl6180x.py

```diff
@@ -109,7 +119,21 @@ def range(self):
         self._write_8(_VL6180X_REG_SYSTEM_INTERRUPT_CLEAR, 0x07)
         return range_
 
-    def read_lux(self, gain):
+    @property
+    def offset(self) -> int:
+        """Read and sets the manual offset for the sensor, in millimeters"""
+        return self._offset
+
+    @offset.setter
+    def offset(self, offset: int) -> None:
+        if not -128 <= offset <= 127:
+            raise ValueError("Offset out of range (-128 ... 127)")
+        if offset < 0:
+            offset = ~(abs(offset) - 1)
+        self._write_8(_VL6180X_REG_SYSRANGE_PART_TO_PART_RANGE_OFFSET, offset)
```

Comment:
Please use `struct.pack` here. Can also let it do the range checking. For example, all this could be:
```python
         self._write_8(_VL6180X_REG_SYSRANGE_PART_TO_PART_RANGE_OFFSET, struct.pack("b", offset)[0])
```
From AN4545:
![image](https://user-images.githubusercontent.com/8755041/143063101-4678428b-9580-449a-9fd0-3ff68d186aea.png)

Reply:
Ah, that looks much simpler and cleaner, thanks!